src/gmail.js

```diff
@@ -1436,8 +1454,7 @@ var Gmail = function(localJQuery) {
                     //TODO : need a refactoring
                     const fd_email_content_html = api.tools.parse_fd_embedded_json_content_html(fd_email);
 
-                    //TODO
-                    const fd_attachments = api.tools.parse_fd_attachments(fd_email["12"]);
+                    const fd_attachments = api.tools.parse_fd_embedded_json_attachments(fd_email["12"]);
```

Comment:
How compatible is this code? Shouldn’t we have fallback to the old code, if the new code fails?

Reply:
I see no reason for this, since previous code was added just as a stub, and was marked with TODO. We have the same code, as I've added here, in production for more than a year and it works fine.
I've also extended tests for all parsing cases to check for parsed attachments, they pass fine as well.

Reply:
Ah right. Fair enough!